pkg/controllers/provisioning/v1alpha1/reallocator/filter.go

```diff
@@ -17,6 +17,9 @@ package reallocator
 import (
 	"context"
 	"fmt"
+	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
+	nodeUtil "github.com/awslabs/karpenter/pkg/utils/node"
```

Comment:
This is a great example of where I screwed up a package name, causing conflicts with commonly named variables. Any ideas for a replacement name to avoid this nasty import renaming issue?

Reply:
Come to think of it, it's not too offensive if you simply name this nodeutils, following standard go package conflict resolution (concatenation). You can see this in main with v1alpha1provisioning, etc

Reply:
I think renaming the package to `nodeUtils` could work! I'll make that change.